README.md

```diff
@@ -34,7 +34,7 @@ regarding Chainlink social accounts, news, and networking.
 
 1. [Install Go 1.21](https://golang.org/doc/install), and add your GOPATH's [bin directory to your PATH](https://golang.org/doc/code.html#GOPATH)
    - Example Path for macOS `export PATH=$GOPATH/bin:$PATH` & `export GOPATH=/Users/$USER/go`
-2. Install [NodeJS v20](https://nodejs.org/en/download/package-manager/) & [pnpm v8 via npm](https://pnpm.io/installation#using-npm).
+2. Install [NodeJS v20](https://nodejs.org/en/download/package-manager/) & [pnpm v9 via npm](https://pnpm.io/installation#using-npm).
```

Comment:
maybe best to remove the version string all together? then we don't have to update it each time

Reply:
It changes maybe once a year, for which I'm willing to update the docs